sktime/utils/datetime.py

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+"""
+pandas.datetime functions related utilities.
+
+modified the _get_freq function
```

Comment:
the file docstring should describe the contents of the file, not the change you made (that goes in the commit message)

Reply:
Sure. Will change it.